packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -84,4 +92,232 @@ export class Copy {
+                    {
+                        throw new ImperativeError({ msg: ZosFilesMessages.datasetCopiedAbortedTargetNotPDSMember.message });
+                    }
+                }
+            }
+
+            /**
+            * If this is a PDS and it exists, verify if the member also exists.
+            */
+            if(targetMember != undefined && targetFound == true){
+                const TargetMemberList = await List.allMembers(targetSession, targetDataset,
+                    {attributes: true, maxLength: 1, start: targetDataset, recall: "wait", pattern: targetMember});
+                if(TargetMemberList.apiResponse.returnedRows > 0){
+                    targetMemberFound = true;
+                }
+            }
+
+            /**
+             *  Check to see if the target dataset exists and if the attributes match the source
+             */
+            if(targetFound == false){
+                /**
+                 *  Create the target dataset if it does not exist based on the source dataset values
+                 */
+                const createOptions = Copy.generateDatasetOptions(options, sourceDataSetObj);
+                /*
+                * If this is a PDS but the target is the sequential dataset and does not exist,
+                * create a new sequential dataset with the same parameters as the original PDS.
+                */
+                if((createOptions.dsorg == "PO" || createOptions.dsorg == "POE") && targetMember == undefined){
+                    createOptions.dsorg ="PS";
+                    createOptions.dirblk = 0;
+                }
+                else if(targetMember != undefined &&  (createOptions.dsorg != "PO" && createOptions.dsorg != "POE"))
+                {
+                    createOptions.dsorg ="PO";
+                    createOptions.dirblk = 1;
+                }
+                await Create.dataSet(targetSession, CreateDataSetTypeEnum.DATA_SET_CLASSIC, targetDataset, createOptions);
+            }
+            else{
+                if(overwriteTarget == undefined && targetMember == undefined){
+                    if (options.promptFn != null) {
+                        overwriteTarget = await options.promptFn(targetDataset);
+                    }
+                }
+                else{
+                    if(overwriteTarget == undefined && targetMemberFound == true)
+                    {
+                        if (options.promptFn != null) {
+                            overwriteTarget = await options.promptFn(targetDataset +"(" + targetMember + ")");
+                        }
+                    }
+                }
+            }
+
+            /*
+            *  Don't overwrite an existing dataset or member if overwrite is false
+            */
+            if(overwriteTarget == undefined || overwriteTarget == true || targetFound == false){
```

Comment:
If the default behavior of the API is to _not_ replace files, should we remove this condition? `overwriteTarget == undefined`

Reply:
I removed the check, as it is no longer relevant.